docs/assets/js/main.js

```diff
@@ -75,9 +75,95 @@ function injectFooterCopywrite() {
     footer.innerHTML = `&copy;2008-${new Date().getFullYear()} Expensify, Inc.`;
 }
 
+function openSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'block';
+
+    // Make body unscrollable
+    const yAxis = document.documentElement.style.getPropertyValue('y-axis');
+    const body = document.body;
+    body.style.position = 'fixed';
+    body.style.top = `-${yAxis}`;
+}
+
+// Function to close the sidebar
+function closeSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'none';
+
+    // Make the body scrollable again
+    const body = document.body;
+    const scrollY = body.style.top;
+
+    // Reset the position and top styles of the body element
+    body.style.position = '';
+    body.style.top = '';
+
+    // Scroll to the original scroll position
+    window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
+}
+
+// Function to change SVG viewBox from Google
```

Comment:
Explain why we're changing the SVG in this function

Reply:
Done